web/src/components/Upstream/UpstreamForm.tsx

```diff
@@ -663,6 +663,9 @@ const UpstreamForm: React.FC<Props> = forwardRef(
                   name: formatMessage({ id: 'page.upstream.step.select.upstream.select.option' }),
                   id: '',
                 },
+                {
+                  name: formatMessage({ id: 'page.upstream.step.select.upstream.select.none' }),
+                },
```

Comment:
Hi @juzhiyuan, @Firstsawyou, and @nic-chen 

Could you guide should remove the id field or change it to another value?

Reply:
just remove the id field

Reply:
OK.